package.json

```
@@ -25,7 +25,7 @@
     "test:browser": "testcafe \"chrome --window-size='1920,1080'\" browser-tests/ --live --dev --lang=fi-FI",
     "test:browser:ci": "testcafe \"chrome:headless --disable-gpu --window-size='1920,1080'\" --lang=fi-FI --screenshots path=report takeOnFails=true --video report browser-tests/",
     "test:browser:wsl2win": "testcafe 'path:`/mnt/c/Program Files/Google/Chrome/Application/chrome.exe`' browser-tests/ --live --dev --lang=fi-FI",
-    "typecheck": "vitest typecheck",
+    "typecheck": "tsc typecheck",
```

```suggestion
    "typecheck": "tsc --project ./tsconfig.json --noEmit",
```

Using `"typecheck": "tsc typecheck"`:
```
$ yarn typecheck
yarn run v1.22.21
$ tsc typecheck
error TS6231: Could not resolve the path 'typecheck' with the extensions: '.ts', '.tsx', '.d.ts', '.cts', '.d.cts', '.mts', '.d.mts'.
  The file is in the program because:
    Root file specified for compilation
```

Using `"typecheck": "tsc --project ./tsconfig.json --noEmit"`:
```
$ yarn typecheck
yarn run v1.22.21
$ tsc --project ./tsconfig.json --noEmit
Done in 16.12s.
```